### ISSUE TITLE
Error importing database dump files

### ISSUE THREAD

**Issue by fabiocontiero** [kool-dev/kool#388]
**Describe the bug**

I'm getting the following error:

`ERROR 2006 (HY000) at line 1: MySQL server has gone away`

That error occurs when I'm trying import a dump file using the following command:

`kool run mysql myDatabase < myDump.slq`

onde `mysql => kool exec database mysql -uroot -p$DB_PORTAL_PASSWORD`

The error doesn't occurs when I'm run:

`docker-compose exec -T database mysql --max_allowed_packet=64M --connect-timeout=60 -uroot -p$DB_PORTAL_PASSWORD myDatabase < myDump.slq`

```
kool version 1.14.0
OS: Ubuntu 20.04.2 LTS
Docker version 20.10.6, build 370c289
docker-compose version 1.26.1, build f216ddbf

kool info:
KOOL_DEPLOY_DOMAIN=completa-web-develop.kool.dev
KOOL_NAME=web
KOOL_GLOBAL_NETWORK=kool_global
KOOL_ASUSER=1000
```

**Comment by fabriciojs** [comment]
@fabiocontiero - please, can you confirm you can consistently reproduce this issue? Everytime you attempt to do it using `kool run mysql < `?

Then please, can you add a pastebin or forward me somehow, the whole output of running this with `--verbose` flag?

**Comment by fabriciojs** [comment]
@fabiocontiero I can somewhat reproduce the issue.

Please can you test the very same command you are trying, but putting it within a script in `kool.yml`? Just to make sure the same command works when executed from there.

**Comment by fabriciojs** [comment]
PR is under review, the fix will go into next release.

**Comment by fabriciojs** [comment]
This has been fixed within our [latest release 1.15.0](https://github.com/kool-dev/kool/releases/tag/1.15.0).
